moduleroot/CONTRIBUTING.md.erb

```diff
@@ -174,6 +174,18 @@ and so on. rspec tests may have the same kind of dependencies as the
 module they are testing. While the module defines in its [Modulefile](./Modulefile),
 rspec tests define them in [.fixtures.yml](./fixtures.yml).
 
+To run specific tests, use the spec test file name and a filter like:
+
+```shell
+bundle exec rspec spec/classes/foreman_spec.rb -e 'should restart passenger'
+```
+
+To run OS specific tests:
```

Comment:
https://github.com/mcanevet/rspec-puppet-facts#running-your-tests describes the `SPEC_FACTS_OS` variable. Note you can also leave of things at the end, so `redhat` will test both 7 and 8 if they're in metadata.json.

Reply:
nice, this will be very useful.  I added this example to the doc too.

Reply:
Part of the note was linking to the gem responsible. Knowing we use that can be very useful.

Reply:
> Part of the note was linking to the gem responsible. Knowing we use that can be very useful.

I don't understand what this means, can you please explain?

Reply:
We rely quite heavily on the gem rspec-puppet-facts. A link to that can be useful.

Reply:
Thank you for the clarification. I updated links to rspec-puppet-facts now.